### DISCUSSION TITLE
[metric-reporter] List of all available metrics

### DISCUSSION THREAD

**Discussion by nlamirault** [airbytehq/airbyte#36313] [Connector Questions]
### Topic

prometheus / opentelemetry metrics

### Relevant information

Could you add to the documentation a list of all of metrics exposed by Airbyte. I would like to create a Grafana dashboard and it is not really easy to find these metrics.

What do you think to add `airbyte` as a Prometheus namespace ? Like that, all of Airbyte's metrics could be something like that `airbyte_xxxx` , and it could be more useful to retrieve these metrics.

A very common metric is also the `build_info` metric to display some information about the application.
Ex: `airbyte_build_info{version="x.y.z", .....}`

**Comment by EJSohn** [comment]
+1

[a comment by an automated account is not shown]
